src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatCompletion.cs

```diff
@@ -86,5 +86,5 @@ public ChatMessage Message
 
     /// <inheritdoc />
     public override string ToString() =>
-        Choices is { Count: > 0 } choices ? string.Join(Environment.NewLine, choices) : string.Empty;
+        string.Join(Environment.NewLine, Choices);
```

Comment:
Assuming individual choices contain multi-line content, wouldn't this make it difficult to distinguish between them?

Reply:
Yeah. I could add a prefix when there are multiple, e.g.
```
Choice 0: ...
Choice 1: ...
```
or something like that.

Reply:
I like that better. Perhaps also consider indenting individual choices if multi-line content?